### DISCUSSION TITLE
webdav网盘pikpak，按照教程设置了root folder id，但成功挂载之后依旧把所有文件夹显示出来了

### DISCUSSION THREAD

**Discussion by githubgj** [AlistGo/alist#1103] [Q&A]
![1](https://user-images.githubusercontent.com/102462164/168474700-a3d6ecff-b368-4ba7-b740-66cf7c1544c2.png)
![2](https://user-images.githubusercontent.com/102462164/168474707-0bc76131-cff6-4e2b-ac86-9f9c998335ac.png)
在此特别感谢作者的付出，但我在用作者的alist成功挂载了PIKPAK，虽然在root folder id设置了特定一个文件夹的ID，但是依旧是把根目录下的所有文件夹都显示出来了，请教作者是什么原因呢？拜谢

**Comment by xhofe** [comment]
https://alist-doc.nn.ci/docs/common-question#%E4%B8%8A%E4%BC%A0%E7%9A%84%E6%96%87%E4%BB%B6%E4%B8%8D%E6%98%BE%E7%A4%BA%E5%88%A0%E9%99%A4%E7%9A%84%E6%96%87%E4%BB%B6%E8%BF%98%E5%9C%A8%E4%BF%AE%E6%94%B9%E6%A0%B9%E7%9B%AE%E5%BD%95%E4%B8%8D%E7%94%9F%E6%95%88
